packages/bigarray-overlap/bigarray-overlap.0.1.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+name:         "bigarray-overlap"
+maintainer:   "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors:      "Romain Calascibetta <romain.calascibetta@gmail.com>"
+homepage:     "https://github.com/dinosaure/overlap"
+bug-reports:  "https://github.com/dinosaure/overlap/issues"
+dev-repo:     "git+https://github.com/dinosaure/overlap.git"
+doc:          "https://dinosaure.github.io/overlap/"
+license:      "MIT"
+synopsis:     "Bigarray.overlap"
+description: """A minimal library to know that 2 bigarray share physically the same memory or not."""
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+  [ "sh" "-exc" "echo \"xen_linkopts = \\\"-l:stubs/xen/liboverlap_stubs_xen_stubs.a\\\"\" >> _build/default/META.overlap"]
+  [ "sh" "-exc" "echo \"freestanding_linkopts = \\\"-l:stubs/freestanding/liboverlap_stubs_freestanding_stubs.a\\\"\" >> _build/default/META.overlap"]
+  [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
+]
+
+install: [
+  [ "dune" "install" "-p" name ] {with-test}
+  [ "./test/test_runes.ml" ] {with-test}
+]
```

Comment:
why is there a special install field for tests? Especially, what is the purpose of the `dune install`?